src/test/java/org/apache/commons/text/lookup/ScriptStringLookupTest.java

```diff
@@ -58,4 +58,17 @@ public void testOne() {
         Assertions.assertEquals("Hello World!", ScriptStringLookup.INSTANCE.lookup("javascript:\"Hello World!\""));
     }
 
+    @Test
+    public void testScriptWithColumn() {
```

Comment:
I think we could rewrite the name of this test to `testScriptUsingMultipleColons` and the other method `testScriptMissingColon` ?

Reply:
Done